specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-01-31-preview/types/applications.json

```diff
@@ -0,0 +1,360 @@
+          "description": "ARM id of the platform resource on which application is deployed"
+        }
+      },
+      "description": "Platform on which application is/will be deployed",
+      "required": [
+        "resourceId"
+      ]
+    },
+    "metadata": {
+      "type": "object",
+      "properties": {
+        "containingFunction": {
+          "$ref": "#/definitions/containingFunctionProperties",
+          "description": "Containing function which includes the call to faultable function"
+        },
+        "faultableFunction": {
+          "$ref": "#/definitions/faultableFunctionProperties",
+          "description": "function in which fault is injected"
+        },
+        "source": {
+          "$ref": "#/definitions/sourceProperties",
+          "description": "defines the location of the code in the class"
+        }
+      }
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Provisioning",
+        "Updating",
+        "Deleting",
+        "Accepted"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "UserAssignedManagedIdentityProperties": {
+      "type": "object",
+      "properties": {
+        "clientId": {
+          "type": "string",
+          "description": "client id of the managed identity"
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "active directory tenant id in which managed identity is provisioned"
+        },
+        "type": {
+          "type": "string",
+          "description": "type of managed identity i.e AzureManagedIdentity"
+        }
+      },
+      "description": "User assigned managed identity assigned to the platform on which application is deployed",
+      "required": [
+        "clientId",
+        "tenantId",
+        "type"
+      ]
+    },
+    "containingFunctionProperties": {
+      "type": "object",
+      "properties": {
+        "signature": {
+          "$ref": "#/definitions/signatureProperties",
+          "description": "Method signature of the containing function"
+        }
+      },
+      "description": "Defines the signature of the function calling the faultable function",
+      "required": [
+        "signature"
+      ]
+    },
+    "faultableFunctionProperties": {
+      "type": "object",
+      "properties": {
+        "signature": {
+          "$ref": "#/definitions/signatureProperties",
+          "description": "Method Signature of the faultable function"
+        },
+        "category": {
+          "type": "string",
+          "description": "SDK or Custom library in which fault is injected"
+        },
+        "faultCapabilities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/faultCapabilityProperties"
+          },
+          "description": "faults supported by torch for the function"
```

Comment:
>torch [](http://example.com/codeflow?start=46&length=5)

Not being domain expert, I'm confused about what torch is, would it be helpful to call it by some other name, or should I just understand torch?

Reply:
Users using this service know what torch is , so it would be better providing it in the description.